### DISCUSSION TITLE
Robbed

### DISCUSSION THREAD

**Discussion by Scnjt15** [community/community#141942] [New to GitHub]
### Select Topic Area

Question

### Body

So I bought a bunch of Bome tokens but it says I have zero balance n idk wat to do can someone help me plz
![IMG_2179](https://github.com/user-attachments/assets/cca7b72e-182b-42dc-85e6-b61fb1b53537)

**Comment by Berchez** [comment]
Why are you asking on github?

**Reply by Scnjt15** [reply]
Idk it's where the Bome site took me to
